microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -153,56 +195,78 @@ The CIDR notation for grouping IP addresses was removed from the `clusterNetwork
+* Previously, after changing the IP adddress, previous `kubeconfig` files for the previous IP address remained. These `kubeconfig` files were unusable because they targeted an IP address not used by {microshift-short}. With this update, the `kubeconfig` files associated with the previous IP address are deleted when the IP address changes. (link:https://issues.redhat.com/browse/OCPBUGS-15740[*OCPBUGS-15740*])
 
-* Previously, installing EC builds of {microshift-short} on {op-system-base-full} 9.2 required OpenvSwitch 2.17, but {op-system} 9.2 includes OpenvSwitch 3.x by default. With this update, {microshift-short} works with OpenvSwitch 3.x on {op-system} 9.2 and later. (link:https://issues.redhat.com/browse/OCPBUGS-11538[*OCPBUGS-11538*])
+* Previously, the `microshift show-config` command did not show configuration changes. With this update, the behavior of the `microshift show-config` command shows the current configuration by default. (link:https://issues.redhat.com/browse/OCPBUGS-13221[*OCPBUGS-13221*])
 
-* Previously, the command `microshift show-config` was prevented from reporting the correct `memoryLimitMB` for etcd. With this update, `microshift show-config` reports the correct `memoryLimitMB` for etcd. (link:https://issues.redhat.com/browse/OCPBUGS-11734[*OCPBUGS-11734*])
+* Previously, KCM and KAS did not exit after getting a cancellation signal. With this update, {microshift-short} shuts down as expected. (link:https://issues.redhat.com/browse/OCPBUGS-18773[*OCPBUGS-18773*])
 
-* Previously, OVN-Kubernetes failed to run on a disconnected {microshift-short} instance. With this update, the OVN-Kubernetes configuration supports running {microshift-short} on hosts with no default route. (link:https://issues.redhat.com/browse/OCPBUGS-11967[*OCPBUGS-11967*])
+* Previously, kubeconfigs used a certificate authority (CA) bundle that included 3 different signers: service network, localhost, and external. This made kubeconfigs interchangeable and capable of validating for other networks. With this update, the certificate authorities used to generate kubeconfig files for {microshift-short}'s embedded components were reconfigured to ensure the kubeconfigs are independent. (link:https://issues.redhat.com/browse/OCPBUGS-8301[*OCPBUGS-8301*])
+
+[discrete]
+[id="microshift-4-14-networking-bug-fixes"]
+==== Networking
+
+* Previously, mixing upper and lower case letters in the hostname makes {microshift-short} fail. This is because the node name was taken directly from the hostname, and Kubernetes does not allow upper case names. With this update, the hostname only uses lower case host names. (link:https://issues.redhat.com/browse/OCPBUGS-8411[*OCPBUGS-8411*])
```

Comment:
```suggestion
* Previously, mixing uppercase and lowercase letters in the hostname caused {microshift-short} to fail. This was because the node name was taken directly from the hostname, and Kubernetes does not use uppercase names. With this update, the hostname only uses lowercase host names. (link:https://issues.redhat.com/browse/OCPBUGS-8411[*OCPBUGS-8411*])
```